firmware/TMS/mcal/stm32f767/.mxproject

```diff
@@ -0,0 +1,19 @@
+[PreviousLibFiles]
```

Comment:
Is this something we want to have pushed to git?

Reply:
It's generated from the ioc, we should decide on Thursday how much to commit and how much to generate locally